lib/cmd/license-file-handler.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const fs = require('fs');
+
+class LicenseFileHandler
+{
+  static getFile(filename)
+  {
+    return new Promise((resolve,reject) => {
+      if (fs.existsSync(filename))
+        return resolve(fs.readFileSync(filename, 'utf8'));
+      else
+        return reject(
+          {
+            'errorCode': 'LICENSE_CONFIG_NOT_FOUND',
+            'errorMessage': 'Could not find any license config file.'
+          });
+    });
+  }
+
+  static parseFile(arg)
+  {
+    return new Promise((resolve,reject) => {
+      let object = null;
+      
+      try {
+        object = JSON.parse(arg);
+      } catch (e) {
+        return reject(
+          {
+            'errorCode': 'LICENSE_CONFIG_CORRUPTED',
+            'errorMessage': 'License config file is not a valid JSON.'
+          });
+      }
+
+      if (object.header !== null && object.header !== undefined &&
```

Comment:
Too big if. 
Please change it to a boolean method  and named it as "isValidJsonObject(object)"/"isValidLicenseJson(object)" or something similar